Jenkinsfile

```diff
@@ -30,7 +30,13 @@ pipeline {
                 sh """mvn -B -V -ff -e clean install -PQuickBuild \
                 -Djavax.net.ssl.trustStore=${env.JAVA_HOME}/jre/lib/security/cacerts \
                 -Djavax.xml.accessExternalSchema=all -Dbuild.number=${payaraBuildNumber} \
-                -Dsurefire.rerunFailingTestsCount=2"""
+                -Dsurefire.rerunFailingTestsCount=2
+                -Dsonar.pullrequest.base=master
+                -Dsonar.pullrequest.branch=${env.ghprbAuthorRepoGitUrl}
```

Comment:
I'm not sure this correct in hindsight.
This would return `https://github.com/MarkWareham/Payara.git`

I think you want:
`${env.ghprbPullAuthorLogin}:${env.ghprbSourceBranch}`

The above would return: `MarkWareham:master`

No idea if you have to escape the ":"